rpc/server_auth.go

```diff
@@ -360,22 +360,44 @@ func (ss *simpleServer) ensureAuthed(ctx context.Context) (context.Context, erro
 		}
 	}
 	if !audVerified {
+		// TODO cleanup at a later date -- urgent issue
 		audVerified = true
-		ss.logger.Infof("hack %v %v", claims.RegisteredClaims, ss.authAudience)
+		ss.logger.Infow("hack %v %v",
+			"registeredClaims", claims.RegisteredClaims,
+			"authAudience", ss.authAudience)
```

Comment:
<img width="1005" alt="Screenshot 2024-03-16 at 11 20 02 AM" src="https://github.com/viamrobotics/goutils/assets/4631633/e925edda-5e9f-4c7c-8a0c-7d1018dee65e">

not sure how useful this line given the image above but not going to take it out in case it ends up being helpful

Reply:
I’d leave it all in until we fix and fully understand